frontend/ui/widget/bookstatuswidget.lua

```diff
@@ -254,7 +256,8 @@ function BookStatusWidget:setStar(num)
 
     table.insert(self.stars_container, stars_group)
 
-    UIManager:setDirty(nil, "ui")
+    -- Individual stars are Buttons, which'll trigger a flash on their own, we want to refresh the group *after* that...
+    UIManager:scheduleIn(0.5, function() UIManager:setDirty(self.stars_container, "ui", nil, true) end)
```

Comment:
@NiLuJe While looking over it quickly (not in detail) only this jumps out, but given the context it can't be too bad. ;-)

Reply:
Yeah, AFAICT, it's a bit of a race condition...

I'll double-check how it behaves with a tickAfterNext, though ;). (With a bit of luck, that'll get merged with the one from Button).

Reply:
Huh, turned out to be fairly broken, in various respects ;D.